python/treelite/frontend.py

```diff
@@ -379,6 +379,13 @@ def from_xgboost(cls, booster):
                                 '`xgboost.Booster` object') from e
         if not isinstance(booster, xgboost.Booster):
             raise ValueError('booster must be of type `xgboost.Booster`')
+        if parse_version(xgboost.__version__) >= parse_version('1.0.0'):
+            # TODO(hcho3): Remove this line once XGBoost implements a method to save JSON model
+            # in-memory. (Right now, save_model() always saves to the disk.)
+            model_json_str = booster.__getstate__()['handle'].decode('utf-8')
+            return cls.from_xgboost_json(model_json_str)
```

Comment:
Does 'from_xgboost_json()' get tested for multiple xgboost versions?

Reply:
For now it is tested with the latest version.